### PR TITLE
fix(docs): use named exports

### PR DESCRIPTION
**Pull request by Ian-Balijawa**
## Related Issues

Fixes #.

## Check List

- [ ] `yarn run prettier` for formatting code and docs

### CODE REVIEW

**Comment by Ian-Balijawa** [comment]
The main docs here https://docs.pmnd.rs/zustand/getting-started/introduction still use default exports.
This at one moment misled me while browsing through. So, I created this PR that includes the named export way.

File: docs/getting-started/introduction.mdx [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: Introduction
 description: How to use Zustand
 nav: 0
 ---
-
+import { create } from 'zustand'
```
Comment (dai-shi) [inline docs/getting-started/introduction.mdx:6]: mistake?
```suggestion

```

File: docs/migrations/migrating-to-v4.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ of the test files in the Zustand repository from v3 to v4.
 **Applicable imports**
 
 ```ts
-import create from 'zustand'
+import { create } from 'zustand'
```
Comment (dai-shi) [inline docs/migrations/migrating-to-v4.md:26]: this shouldn't be changed as it describes v3 api.

```suggestion
import create from 'zustand'
```

File: docs/previous-versions/zustand-v3-create-context.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ which avoids misusing the store hook.
 > **Note**: This function is deprecated in v4 and will be removed in v5. See [Migration](#migration).
 
 ```jsx
-import create from 'zustand'
+import { create } from 'zustand'
```
Comment (dai-shi) [inline docs/previous-versions/zustand-v3-create-context.md:12]: this should be changed, as it describes about v3 api.

**Review by dai-shi — COMMENTED** [review]
those look like intentional ones.
do you find anything else to fix?

File: docs/previous-versions/zustand-v3-create-context.md [hunk 4]
```diff
@@ -83,7 +83,7 @@ export default function App() {
 ## createContext usage with initialization from props
 
 ```tsx
-import create from 'zustand'
+import { create } from 'zustand'
```
Comment (dai-shi) [inline docs/previous-versions/zustand-v3-create-context.md:86]: same for this.
```suggestion
import create from 'zustand'
```

**Comment by sewera** [comment]
After @dai-shi's reviews it means that this PR should be declined if I understood correctly.

**Comment by dai-shi** [comment]
@Ian-Balijawa concerned with https://docs.pmnd.rs/zustand/getting-started/introduction , but this PR doesn't include such change.
Let's wait a bit and without further commits, close it.

**Comment by dai-shi** [comment]
Oh, okay, #1559 already addressed it. Closing.
Thanks all!

**Pull request closed without merging**